### ISSUE TITLE
UI Accessibility - Page, header, and image titles and text

### ISSUE THREAD

**Issue by Worfle** [solo-io/gloo-portal-issues#201]
From #196 ::

- [x] Dynamic page title
  - http://portal.demo.solo.io/apis/ - "E-commerce Portal - APIs"
  - http://portal.demo.solo.io/apis/gloo-system/petstore?v=v1 - "E-commerce Portal - petstore v1"
  - http://portal.demo.solo.io/apis/gloo-system/petstore?v=checked - "E-commerce Portal - petstore checked"
  - http://portal.demo.solo.io/keys/ - "E-commerce Portal - API Keys"
- [x] Alt text for all images and SVGs
- [x] Missing headings
  - All pages need at least 1 `<h1>` tag, we can swap whatever `div` contains the page title to use a `h1`

**Comment by Worfle** [comment]
To add context, @kcbabo  and @williamgrh , I'm reading these and seeing this set as solely relevant to PortalUI.  Altering page title and forcing h1s in seem odd in an app the style of Admin, but I can if we need (may need to discuss the < h1 > concept)

I'm adding the text for images/svgs to Admin as well, as it's just good practice.

**Comment by kcbabo** [comment]
Primary focus here was the dev portal UI, so approach sounds good to me.
